dev/large-requirements.txt

```diff
@@ -1,3 +1,20 @@
 ## Large test reqs
 matplotlib
 tf2onnx
+
+# Required by mlflow.keras
+keras==2.3.1
```

Comment:
It's a bit confusing since we've refactored our dependency management for tests, but all of these currently reside in https://github.com/mlflow/mlflow/blob/master/dev/extra-ml-requirements.txt and are installed before large tests are executed. We can remove them from this file.

Reply:
interesting, the tests were complaining before that tensorflow did not exist before i added it to the small-requirements file.

also, it seems like that file has the wrong version of spark for the tests, I think we need 3.0.0